validation/translations.rst

```diff
@@ -123,6 +123,29 @@ Now, create a ``validators`` catalog file in the ``translations/`` directory:
 You may need to clear your cache (even in the dev environment) after creating
 this file for the first time.
 
+You can also use :class:`Ns\\TranslatableMessage` to build your violation message::
```

Comment:
```suggestion
You can also use :class:`Symfony\\Component\\Translation\\TranslatableMessage` to build your violation message::
```

`Ns` in my example was just a placeholder for the real namespace :slightly_smiling_face:

Reply:
Ok
I though you gave a shortcut or something